packages/nextra/src/client/components/search.tsx

```diff
@@ -184,22 +194,21 @@ export const Search: FC<SearchProps> = ({
   const handleChange = useCallback(
     (event: SyntheticEvent<HTMLInputElement>) => {
       const { value } = event.currentTarget
-      onChange(value)
+      setSearch(value)
     },
-    [onChange]
+    []
   )
 
   const handleSelect = useCallback(
-    async (searchResult: PagefindResult | null) => {
+    (searchResult: PagefindResult | null) => {
       if (!searchResult) return
       // Calling before navigation so selector `html:not(:has(*:focus))` in styles.css will work,
       // and we'll have padding top since input is not focused
       inputRef.current?.blur()
-      await router.push(searchResult.url)
-      // Clear input after navigation completes
-      onChange('')
+      router.push(searchResult.url)
+      setSearch('')
```

Comment:
removing the async/await syntax since `router.push` from `next/navigation` isn't supported

Reply:
Oh nice, maybe we should enable some eslint rule to remove unneeded async await ❤️

Reply:
```text
/Users/dmytro/Desktop/nextra/packages/nextra/src/client/components/search.tsx
  198:7  error  Unexpected `await` of a non-Promise (non-"Thenable") value  @typescript-eslint/await-thenable
```
I will push in your PR